src/main/java/Timo/Timo.java

```diff
@@ -0,0 +1,584 @@
+package Timo;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+import java.util.*;
+import java.io.File;
+import java.time.format.DateTimeFormatter;
+
+class Task {
+    private String taskype = "T";
+    private boolean mark;
```

Comment:
Is there a typo in ```String taskype```? Perhaps ```boolean mark``` could sound more like a boolean, for example adding a prefix such as is, can, has etc.

Reply:
Okay thanks for pointing out my mistakes. I will go and resolve them!